### DISCUSSION TITLE
Something down the lines of BurpBounty(burp extension with nuclei like templates)

### DISCUSSION THREAD

**Discussion by jakre1234** [projectdiscovery/nuclei#2674] [Q&A] [answered]
Hi Team,
Just wondering if ProjectDiscovery have any plans to implement nuclei as an BurpSuite extension something what BurpBounty does(https://burpbounty.net/). This will be really useful, if we have passive and active template based checks within burp 's inscope history itself.
Maybe nuclei or maybe a completely separate tool/extension for this.

Thanks and Regards,

**Comment by ehsandeep** [comment, marked as answer]
@jakre1234 not sure you are aware of https://github.com/projectdiscovery/nuclei-burp-plugin which currently supports creating nuclei template out of burp request history, and we intend to support more features, including scanning as listed down in the issue section - https://github.com/projectdiscovery/nuclei-burp-plugin/issues but if you specific feature request suggestion, feel free create a new issue on the same project.

**Reply by ayadim** [reply]
i hope you add drop-down list to scan a  request in history with a nuclei template i want i will be great movment

**Reply by jakre1234** [reply]
Hey @ayadim , it's already there. You can scan any URL from your burp history directly into the nuclei plugin for burp. The only issue with this is that it supports one URL at a time. I would love to see multiple URL support same as we have on CLI.
